homeassistant/components/anglian_water/sensor.py

```diff
@@ -0,0 +1,58 @@
+"""Sensor platform for anglian_water."""
+
+from __future__ import annotations
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import AnglianWaterDataUpdateCoordinator
+from .entity import AnglianWaterEntity
+
+SENSORS = (
+    SensorEntityDescription(
+        key="anglian_water_previous_consumption",
+        name="Previous Consumption",
+        icon="mdi:water",
```

Comment:
Please use icon and entity name translations here

Reply:
I just need to do this now I think.